### PR TITLE
added game over screen just like high score screen

### PR DESCRIPTION
**Pull request by Rishabh32700**

### CODE REVIEW

**Review by idna001 — CHANGES_REQUESTED** [review]
See comments, and maybe you can solve this one new issue that sonarcloud added: [LINK](https://sonarcloud.io/project/issues?id=idna001_Couple-Memory&pullRequest=100&issueStatuses=OPEN,CONFIRMED&sinceLeakPeriod=true)

File: src/App.js
[the remark this answers is not in the document]
Comment (Rishabh32700) [reply src/App.js:1]: okay okay leme solve look into this

[the remark this answers is not in the document]
Comment (Rishabh32700) [reply src/App.js:1]: i have updated the PR just test it now also the sonar error is resolved plz verify

**Review by idna001 — APPROVED** [review]
Now it's perfect, thank you! Happy Hacktober!

**Comment by Rishabh32700** [comment]
plz add hacktober-accepted label

**Pull request merged**